apps/web/src/pages/templates/workflow/digest/TimedDigestWillBeSentHeader.tsx

```diff
@@ -51,6 +51,12 @@ const getOrdinal = (num: string | number) => {
   return num + nth;
 };
 
+const WEEKDAYS_ORDER: string[] = ['monday', 'tuesday', 'wednesday', 'thursday', 'friday', 'saturday', 'sunday'];
+
+const sortWeekdays = (weekdays: string[]): string[] => {
+  return weekdays.sort((a, b) => WEEKDAYS_ORDER.indexOf(a) - WEEKDAYS_ORDER.indexOf(b));
+};
```

Comment:
the selected weekdays will be sorted when showing in the "Will be sent" field label